docs/source/documentation/fibertube_tracking.rst

```diff
@@ -0,0 +1,335 @@
+``scil_fibertube_tracking.py``, you are able to track without relying on
+a discretized grid of directions or fODFs. Instead, you will be
+propagating a streamline through fibertubes and controlling the
+resolution by using a ``blur_radius``. The way it works is as follows:
+
+Seeding
+~~~~~~~
+
+For now, a number of seeds is set randomly within the first segment of
+every fibertube. We can however change the number of fibertubes that
+will be tracked, as well as the amount of seeds within each. (See
+Seeding options in the help menu).
+
+Tracking
+~~~~~~~~
+
+When the tracking algorithm is about to select a new direction to
+propagate the current streamline, it will build a sphere of radius
+``blur_radius`` and pick randomly from all the fibertube segments
+intersecting with it. The larger the intersection volume, the more
+likely a fibertube segment is to be picked and used as a tracking
+direction.
+
+
+.. image:: https://github.com/user-attachments/assets/0308c206-c396-41c5-a0e1-bb69b692c101
+   :alt: Visualization of the blurring sphere intersecting with segments
+
+
+For more information and better visualization, watch the following
+presentation: https://docs.google.com/presentation/d/1nRV2j_A8bHOcjGSHtNmD8MsA9n5pHvR8/edit#slide=id.p19
+
+
+This makes fibertube tracking inherently probabilistic.
+Theoretically, with a ``blur_radius`` of 0, any given set of coordinates
+has either a single tracking direction because it is within a fibertube,
+or no direction at all from being out of one. In fact, this behavior
+won't change until the diameter of the sphere is larger than the
+smallest distance separating two fibertubes. When this happens, more
+than one fibertubes will intersect the ``blur_radius`` sphere and
+introduce partial volume effect.
+
+The interface of the script is very similar to
+``scil_tracking_local_dev.py``, but simplified and with a ``blur_radius``
+option. Let us do:
+
+::
+
+   scil_fibertube_tracking.py fibertubes.trk tracking.trk 0.01 0.01 --nb_fibertubes 3 --out_config tracking_config.txt --processes 4 -v -f
+
+This should take around 5 minutes. The loading bar of each thread will
+only update every 100 streamlines. It may look like it's frozen, but it
+rest assured it's still going!
+
+Reconstruction analysis
+~~~~~~~~~~~~~~~~~~~~~~~
+
+By using the ``scil_fibertube_score_tractogram.py`` script, you are able
+to obtain measures on the quality of the fibertube tracking that was
+performed. Here is a description of the computed metrics:
+
+VC: "Valid Connection": A streamline that ended within the final segment
+of the fibertube in which it was seeded.
+
+IC: "Invalid Connection": A streamline that ended in the first or final
+segment of another fibertube.
+
+NC: "No Connection": A streamline that has not ended in the first or final
+segment of any fibertube.
+
+.. image:: https://github.com/user-attachments/assets/4871cb09-313e-499a-b56d-a668bdb631db
+   :alt: Visual representation of VC, IC, and NC
```

Comment:
Like we said this week at our meeting, can you create a version of the images without the dashed white lines? They are more confusing than helping...